subscriptions/models.py

```diff
@@ -313,3 +313,22 @@ class SubscriptionSendFailure(models.Model):
 
     def __str__(self):  # __unicode__ on Python 2
         return str(self.id)
+
+
+@python_2_unicode_compatible
+class EstimatedSend(models.Model):
+
+    """ Estimated number of messages to be sent per message set per day
+    """
+    send_date = models.DateField()
+    messageset = models.ForeignKey(MessageSet, related_name='estimates',
+                                   null=False)
+    estimate = models.IntegerField(null=False, blank=False)
+    estimate_unique = models.IntegerField(null=False, blank=False)
```

Comment:
I'm wondering if maybe we could name these better? Maybe something like `estimate_subscriptions`, and `estimate_identities`?